Modules/CTP/src/CTPTrendingTask.cxx

```diff
@@ -0,0 +1,223 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file     CTPTrendingTask.cxx
+/// \author   Lucia Anna Tarasovicova
+///
+
+#include "CTP/CTPTrendingTask.h"
+#include "CTP/TH1ctpReductor.h"
+
+#include <QualityControl/DatabaseInterface.h>
+#include <QualityControl/MonitorObject.h>
+#include <QualityControl/RootClassFactory.h>
+#include <QualityControl/QcInfoLogger.h>
+#include <QualityControl/ActivityHelpers.h>
+#include <QualityControl/RepoPathUtils.h>
+#include <QualityControl/Reductor.h>
+#include <QualityControl/UserCodeInterface.h>
+#include <CCDB/BasicCCDBManager.h>
+#include <DataFormatsCTP/Configuration.h>
+#include <DataFormatsCTP/RunManager.h>
+
+#include <TCanvas.h>
+#include <TH1.h>
+
+using namespace o2::quality_control;
+using namespace o2::quality_control::core;
+using namespace o2::quality_control::postprocessing;
+using namespace o2::quality_control::repository;
+using namespace o2::quality_control_modules::ctp;
+using namespace o2::ctp;
+
+void CTPTrendingTask::configure(const boost::property_tree::ptree& config)
+{
+  mConfig = TrendingConfigCTP(getID(), config);
+}
+
+void CTPTrendingTask::initialize(Trigger, framework::ServiceRegistryRef services)
+{
+  // Preparing data structure of TTree
```

Comment:
Please delete any existing pointers in `mPlots` and clear it to account for the correct START -> STOP -> START sequence